vault/testdata/Dockerfile

```
@@ -0,0 +1,7 @@
+FROM gcr.io/distroless/static-debian11
```

I wasn't able to get this Docker image to work locally -- it always gives me something like:

```
core: Error determining plugin version:
  error=
  | 1 error occurred:
  | \t* failed to dispense v4 backend plugin: Unrecognized remote plugin message:
  | Failed to read any lines from plugin's stdout
...  
```

However, if I change it to `ubuntu:22.04`, it works fine.

I wonder if it has to do with the fact that this image doesn't have `/bin/sh` by default?
```suggestion
FROM ubuntu:22.04
```

---

From hitting similar issues myself, I wonder if it works with `CGO_ENABLED=0`? If switching the base container is the only step needed to fix, it sounds like the most likely cause is missing libc bindings.

However, I'm not opposed to switching to ubuntu for testing purposes as it is a bit more forgiving.

---

Ah, good call. Doing `CGO_ENABLED=0` did the trick.